wee_slack.py

```diff
@@ -3751,7 +3763,7 @@ def trace_calls(frame, event, arg):
 
             tokens = config.slack_api_token.split(',')
             for t in tokens:
-                s = SlackRequest(t, 'rtm.start', {})
+                s = SlackRequest(t, 'rtm.start', {"batch_presence_aware": True })
```

Comment:
I'd like a method for connecting with `rtm.start` so we don't have to duplicate the options.